app/javascript/components/errors/DefaultErrorPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { ExclamationCircleIcon } from '@heroicons/react/24/outline';
+import {
+  Button, Card, Col, Container, Row,
+} from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+export default function DefaultErrorPage({ error, errorInfo }) {
+  const [open, setOpen] = useState(false);
+  const { t } = useTranslation();
+
+  const issue = {
+    labels: 'v3-alpha,bug',
+    template: 'greenlight-issue.md',
+    title: `Error Message: "${error}"`,
+  };
+
+  const issueUrl = `https://github.com/bigbluebutton/greenlight/issues/new?labels=${issue.labels}&template=${issue.template}&title=${issue.title}`;
```

Comment:
Um, haven't thought about that,
I think since regular users will use it and since beta will be released soon we may like to drop the component stack trace and report functionalities and make a generic message where we indicate that something went wrong and ask users to report the incident to the administrator(s) ?